packages/core/__tests__/interpreter/race.js

```diff
@@ -138,3 +138,25 @@ test('saga race between sync effects', () => {
       expect(actual).toEqual(expected)
     })
 })
+test('saga race cancelling joined tasks', () => {
+  const middleware = sagaMiddleware()
+  applyMiddleware(middleware)(createStore)(() => {})
+
+  function* genFn() {
```

Comment:
Could you rewrite this test (or add another one) to cover this in more explicit manner? The fact that race cancels losers is an implicit one, would be good to have a straightforward test for this.